Lib/unittest/test/testmock/testmock.py

```diff
@@ -1769,6 +1769,35 @@ def test_attribute_deletion(self):
             self.assertRaises(AttributeError, getattr, mock, 'f')
 
 
+    def test_mock_does_not_raise_on_repeated_attribute_deletion(self):
+        # bpo-20239: Assigning and deleting twice an attribute raises.
```

Comment:
No sure we need the bpo comment, what will this mean in 10 years time when we're on a different tracker?

Reply:
We need it precisely to distinguish the tracker. For example, there is a PEP proposing moving to the GitHub issue tracker but the existing issues won't be moved. Therefore, issuexxxx is ambiguous while bpo-xxxx is not.

Reply:
I meant not having the comment at all, what value does the comment add? The test name and its code should be enough :-)

Reply:
@pablogsal - please remove this comment.